src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  todos: Todo[],
+  setFilterStatus: (filter: Filter) => void,
+  isCompleted: boolean,
+  filterStatus: Filter,
+  handleDelete: (id: number) => void,
+};
+
+export const TodoFooter: React.FC<Props> = ({
+  todos,
+  setFilterStatus,
+  isCompleted,
+  filterStatus,
+  handleDelete,
+}) => {
+  const activeTodos = [...todos].filter(todo => !todo.completed);
```

Comment:
`filter` returns new array
```suggestion
  const activeTodos = todos.filter(todo => !todo.completed);
```